app/src/main/java/org/mifos/mobile/viewModels/RegistrationVerificationViewModel.kt

```diff
@@ -0,0 +1,46 @@
+package org.mifos.mobile.viewModels
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.observers.DisposableObserver
+import io.reactivex.schedulers.Schedulers
+import okhttp3.ResponseBody
+import org.mifos.mobile.api.DataManager
+import org.mifos.mobile.models.register.UserVerify
+import javax.inject.Inject
+
+class RegistrationVerificationViewModel @Inject constructor(private val dataManager : DataManager?) : ViewModel() {
```

Comment:
No need to create this class, I believe you could use RegistrationViewModel instead and move the verification logic over there

Reply:
alright, converting it to draft and then will update this after registrationFragment is done with migration.

Reply:
Done